pkg/build/builder/sti.go

```diff
@@ -129,6 +129,10 @@ func makeVolumeSpec(src localObjectBuildSource, mountPath string) s2iapi.VolumeS
 // Build executes S2I build based on configured builder, S2I builder factory
 // and S2I config validator
 func (s *S2IBuilder) Build() error {
+	return s.buildInternal(getContainerNetworkConfig)
+}
+
+func (s *S2IBuilder) buildInternal(getContainerNetworkConfigFunc getContainerNetworkConfigFn) error {
```

Comment:
nit: could you add a comment saying that this function is provided for the tests?